dpd/p4/sidecar.p4

```diff
@@ -1951,6 +1958,31 @@ control Ingress(
 			meta.bridge_hdr.setInvalid();
 			ig_tm_md.bypass_egress = 1w1;
 		}
+
+		if (meta.nat_ingress_hit) {
+			// This works around a few things which cropped up in
+			// supporting several concurrent Geneve options:
+			//
+			// - Why aren't we just accessing
+			//   `hdr.geneve_opts.oxg_ext_tag`?
+			//   > error: bytes within W2 appear multiple times in
+			//     checksum 1
+			//   And so on, for various other checksums. I assume
+			//   this is because there are theoretically several
+			//   parser paths which could set this header. However,
+			//   we know on this code path that the header could only
+			//   have been pushed and was never *in* the initial
+			//   parse (nat_ingress).
+			//
+			// - Why are we storing this in metadata?
+			//   > error: Non-zero constant entry in checksum
+			//     calculation not implemented yet: 16w0x129
+			//   Fairly straightforward, and I imagine this could
+			//   be the easier one to fix in tofino-p4c. This
+			//   `todo!()` covers various other tricks, including
+			//   wrapping the const in a struct/header.
+			meta.nat_ingress_csum = 16w0x0129;
```

Comment:
It's not obvious to me why this is always 0x129.  I believe you that it is, but I could use an explanation here.